packages/backend-core/src/db/couch/DatabaseImpl.ts

```diff
@@ -246,6 +246,35 @@ export class DatabaseImpl implements Database {
     })
   }
 
+  async bulkRemove(documents: Document[], opts?: { silenceErrors?: boolean }) {
+    const response: Nano.DocumentBulkResponse[] = await this.performCall(db => {
+      return () =>
+        db.bulk({
+          docs: documents.map(doc => ({
+            ...doc,
+            _deleted: true,
+          })),
+        })
+    })
+    if (opts?.silenceErrors) {
+      return
+    }
+    let errorFound = false
+    let errorMessage: string = "Unable to bulk remove documents: "
+    for (let res of response) {
+      if (res.error) {
+        errorFound = true
+        errorMessage += res.error
+      }
+    }
+    if (errorFound) {
+      throw new CouchDBError(errorMessage, {
+        name: this.name,
+        status: 400,
```

Comment:
Why 400?

Reply:
Unsure what status code to use - it isn't a 500 as we got a 200 back from the bulk API which lead to this, generally CouchDB uses 400 when its not a 409 and its just due to some sort of user input - this felt like the best code based on the scenario (something in the list couldn't be deleted, but it didn't go totally wrong).